installer/ClientInstaller.cs

```diff
@@ -131,12 +128,12 @@ public class ClientInstaller : MumbleInstall {
 		shortcutDir.Shortcuts = new ExeFileShortcut[] { menuShortcut };
 		desktopDir.Shortcuts = new ExeFileShortcut[] { deskShortcut };
 
-		var binaryFiles = new File[binaries.Length];
+		var binaryFiles = new File[binaries.Capacity];
```

Comment:
Capacity is wrong, you want to use `Count`.

Reply:
```suggestion
		var binaryFiles = new File[binaries.Count];
```

Reply:
The internet has lied to me! :scream: 
I thought that Capacity sounded like the wrong thing to use but after I found that suggested for getting a list's length on two independent websites, I used it anyway xD

Reply:
I was confused why I could not add code suggestions, yet you were able to? Now I'm even more confused…